presto-verifier/src/main/java/com/facebook/presto/verifier/prestoaction/NodeResourceClient.java

```diff
@@ -13,9 +13,7 @@
  */
 package com.facebook.presto.verifier.prestoaction;
 
-import com.facebook.airlift.json.JsonCodec;
-
-public interface PrestoResourceClient
+public interface NodeResourceClient
 {
-    <V> V getJsonResponse(String path, JsonCodec<V> responseCodec);
+    int getSize(String path);
```

Comment:
What does `NodeResourceClient::getSize` mean? As an API this name is very vague.

Reply:
It's the size of the array of objects in the Json response (list of HeartbeatFailureDetector.Stats objects ).  This size tells us the size of the test cluster which is needed to resolve HIVE_TOO_MANY_OPEN_PARTITIONS error that occurs when the  verifier cluster you're using is smaller than the production cluster.

Reply:
Maybe we should give it a better name. The method name should be self-explanatory and self-documenting. We can call it `getClusterSize` and `getWorkerCount`.